docs/operating-system/debugging.md

```diff
@@ -9,21 +9,27 @@ This section is not for end users. End users should use the [SSH add-on] to SSH
 
 [SSH add-on]: https://github.com/home-assistant/hassio-addons/tree/master/ssh
 
-## SSH access to the host
+## SSH access
+
+Home Assistant has two different types of ssh access.
+- End user SSH on port 22. This port is used by the [SSH add-on], it provides easy but limited SSH access to the system.
+- Host access SSH on port 22222. This method provides full access to the host with all privileges.
+
+### Enable SSH host access
 
 :::info
-SSH access through the [SSH add-on] (which will give you SSH access through port 22) will not provide you with all the necessary privileges, and you will be asked for a username and password when typing the 'login' command. You need to follow the steps below, which will setup a separate SSH access through port 22222 with all necessary privileges.
+A public/private key pair is required when setting up SSH host access, see [Generating SSH Keys](#generating-ssh-keys) for help.
```

Comment:
I'm not sure if that should be in the info compared to what was here before?
Why is this now an information block?

Reply:
I removed the info which was there before and moved it to the "SSH access" section above where i explain clearly that there are 2 ssh access methods with different scopes.

Then i added this new info here, because you need a keypair before you start.

Reply:
So the key point of the info shown was to explain that an add-on doesn't have access to these parts. Additionally, it writes about the login procedure (which got lost now).

I don't see how the current addition help or is info worthy?

Reply:
As already said, the "key point" moved one step up where both ssh access types are described now.

What do you mean with login procedure which was lost? That a username and password is asked? That depends fully on your client, if you pre-enter a username, save a password or link a private key file to use. So there is nothing lost.